app/components/inventory/rank-select.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+import SelectValues from 'hospitalrun/utils/select-values';
+
+const c = Ember.computed;
+
+export default Ember.Component.extend({
+  label: 'Rank',
+  rankOptions: ['A', 'B', 'C'],
```

Comment:
lets put this in init or a CP, putting complex values on the prototype is likely to cause grief.